yaml/buildings.yaml

```diff
@@ -106,6 +106,9 @@ filters:
   # address points
   - filter:
       "addr:housenumber": true
+      way:
```

Comment:
This looks to limit it to address nodes only?

What about adding 2nd filter like:

```
- "addr:housenumber": true
- building: true
```